### PR TITLE
Allow passing arguments to shiny::shinyApp()

### PR DESCRIPTION
**Pull request by chasemc**
To allow things like setting port, etc. see: https://github.com/ThinkR-open/golem/issues/128

Potential conflicts:
If any golem options were named one of the following:
onStart
options
enableBookmarking

Happy to discuss this more if needed

### CODE REVIEW

**Comment by ColinFay** [comment]
Thanks @chasemc! 

Nifty idea. 
Got some feedbacks before merging :)

File: inst/shinyexample/R/run_app.R [hunk 1]
```diff
@@ -1,11 +1,23 @@
 #' Run the Shiny Application
 #'
+#' @param onStart optional, described in ?shiny::shinyApp
```
Comment (ColinFay) [inline inst/shinyexample/R/run_app.R]: Can you use `@inheritParams shiny::shinyApp` here ?

File: inst/shinyexample/R/run_app.R [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+#' @param ... arguments to pass to golem_opts
+#'
 #' @export
 #' @importFrom shiny shinyApp
 #' @importFrom golem with_golem_options
-run_app <- function(...) {
+run_app <- function(onStart = NULL,
+                    options = list(), 
+                    enableBookmarking = NULL,
+                    ...) {
```
Comment (ColinFay) [inline inst/shinyexample/R/run_app.R]: We should also have `options` here (from `shinyApp`)

File: inst/shinyexample/R/run_app.R (hunk 1 above)
Comment (chasemc) [reply inst/shinyexample/R/run_app.R]: Fixed!
I was unaware of `@inheritParams `. That is going to save me so much time in the future.

File: inst/shinyexample/R/run_app.R (hunk 2 above)
Comment (chasemc) [reply inst/shinyexample/R/run_app.R]: Not  sure what you mean.  Line 12 there?

**Comment by chasemc** [comment]
Looks like Appveyor job needs to be restarted: 
`500: Internal Server Error`

File: inst/shinyexample/R/run_app.R (hunk 2 above)
Comment (ColinFay) [reply inst/shinyexample/R/run_app.R]: Yep, original shinyApp function has an `options` parameter, we should be able to use it from run_app() too

**Comment by chasemc** [comment]
@ColinFay  can you try restarting appveyor? This may need to be looked at again since it's been awhile

**Pull request merged**